my_calc.py

```diff
@@ -1,8 +1,8 @@
 import numpy as np
 
-x_len = 5
+x_len = int(input('How many arrays of 100? '))  # Not the best error handling!
 y_len = 100
 a = np.random.randint(1, 100, [x_len, y_len])
 
-for i in range(0, 5):
+for i in range(0, x_len):
```

Comment:
Can we use another variable name here?

Reply:
What were you thinking? ii? x_length?

Reply:
Yes, I think x_length would be more clear

Reply:
Done 53acbcb